src/Kernel32.Tests/storebanned/Kernel32Facts.cs

```diff
@@ -1130,6 +1131,49 @@ unsafe uint GetProcessIdThreadProc(void* data)
         GC.KeepAlive(threadStartRoutine); // Make sure that the delegate stays alive until the test is done
     }
 
+    [Fact]
+    public async Task DeviceIOControlAsync_Works()
+    {
+        const uint FSCTL_SET_ZERO_DATA = 0x000980c8;
+        string fileName = Path.Combine(Environment.CurrentDirectory, "test.txt");
```

Comment:
We shouldn't create files based on an assumption of the current directory, since xunit doesn't guarantee a particular current directory.

Reply:
Perhaps create a file under the temp directory (and delete it later)?